projects/storefrontlib/src/cms-structure/routing/suffix-routes/get-suffix-url-matcher.ts

```diff
@@ -0,0 +1,55 @@
+import { UrlMatchResult, UrlSegment } from '@angular/router';
```

Comment:
having 'get-' in the file name looks strange.

Reply:
renamed to `suffix-url-matcher.ts`